CIP-0035/README.md

```diff
@@ -240,10 +261,10 @@ However, this becomes less so as the platform starts to mature and is neither su
 Furthermore, while many changes to Cardano are obscure or not of interest to many community members, there is a much larger community who have a keen interest in changes to Plutus Core: dApp developers. 
 Hence it is especially important to have a clear way for this community to be able to propose changes and see how they are progressing.
 
-### Do removals and changes really need a new language version?
+### Do removals and changes really need a new ledger language?
 
-Not being able to make removals or changes to behaviour without a LV is quite painful. 
-For example, it means that we cannot just fix bugs in the semantics: we must remain bug-for-bug compatible with any given LV.
+Not being able to make removals or changes to behaviour without a LL is quite painful. 
+For example, it means that we cannot just fix bugs in the semantics: we must remain bug-for-bug compatible with any given LL.
 
 It is tempting to think that if we can show that a particular behaviour has never been used in the history of the chain, then changing it is backwards-compatible, since it won’t change the validation of any of the actual chain. 
 However, this is sadly untrue. 
```

Comment:
... also because there could be an un-triggered validator on the chain that relies on the old behaviour?  Maybe the code is law and that's the owner's problem, but they'd have a reasonable grievance if the behaviour changed from what was advertised when the code was written.